rs/consensus/benches/validate_payload.rs

```diff
@@ -372,32 +376,45 @@ fn validate_payload_benchmark(criterion: &mut Criterion) {
 }
 
 fn serialization_benchmark(criterion: &mut Criterion) {
-    let mut group = criterion.benchmark_group("serialization");
-    group.sample_size(30);
+    let mut group = criterion.benchmark_group("ingress_payload_serialization_deserialization");
+    group.sample_size(50);
     group.measurement_time(std::time::Duration::from_secs(10));
 
-    for message_count in (2000..=8000).step_by(2000) {
+    for (message_count, message_size) in [
+        (1_000, 4_000),
+        (2_000, 4_000),
+        (1, 4_000_000),
+        (1, 8_000_000),
+    ] {
         run_test(
-            "serialization_benchmark",
             |now: Time, _: &mut ConsensusPoolImpl, _: &dyn PayloadBuilder| {
                 let seed = CERTIFIED_HEIGHT + PAST_PAYLOAD_HEIGHT + 10;
-                let ingress = prepare_ingress_payload(now, message_count, seed as u8);
-                let name = format!("serialization_{}_kb_payload", message_count);
-                group.bench_function(&name, |bench| {
-                    bench.iter(|| {
-                        let proto: pb::IngressPayload = (&ingress).into();
-                        black_box(proto);
-                    })
-                });
-                let name = format!("deserialization_{}_kb_payload", message_count);
-                group.bench_function(&name, |bench| {
-                    let p: pb::IngressPayload = (&ingress).into();
-                    bench.iter(|| {
-                        let proto = p.clone();
-                        let deser: IngressPayload = proto.try_into().unwrap();
-                        black_box(deser);
-                    })
-                });
+                let ingress = prepare_ingress_payload(now, message_count, message_size, seed as u8);
+
+                group.bench_function(
+                    format!("serialization_{}x{}", message_count, message_size),
```

Comment:
```suggestion
                    format!("serialization_{}x{}kB", message_count, message_size),
```

or similar, so its more clear which value is the message count and which one the size

Reply:
good idea, changed